controllers/cloud.redhat.com/namespacereservation_controller.go

```diff
@@ -319,3 +327,17 @@ func hardCodedUserList() map[string]string {
 		"system:authenticated": "Group",
 	}
 }
+
+func parseDurationTime(duration *string) (time.Duration, error) {
```

Comment:
I remember the reason I had done it this way; `res.Spec.Duration` is of time `*string`, so I was passing that in directly to `parseDurationTime` and continued with that type. I'll convert that type to be `string` instead of `*string` when it is passed into `parseDurationTime`.

Reply:
Solution was to dereference the pointer to `res.Spec.Duration` when passing it as a parameter to `parseDurationTIme`.